### ISSUE TITLE
Error when uploading custom theme. No images found in zip file.

### ISSUE THREAD

**Issue by ibaeza1** [t1m0thyj/WinDynamicDesktop#226]
Hi! i was trying to create a custom theme. I downloaded a .heic file and then transformed it in the images i wanted to use. Then i did the .json file. The file looks like this:

{
  "imageFilename": "anime_*.jpeg",
  "imageCredits": "Nacha",
  "sunriseImageList": [
    1
 
  ],
  "dayImageList": [
    1
  ],
  "sunsetImageList": [
    2
  ],
  "nightImageList": [
    3
  ]
}

Then i made a zip file with the json and all the images i wanted to use and tried to upload it.
When i tried to upload it i had this error:
![image](https://user-images.githubusercontent.com/37149485/87827142-3d4edb00-c848-11ea-8fb9-de9045e3c05c.png)

Its weird because all the images are in the zip file, so i dont know how to fix it.
If anyone knows why this happens please tell me :) 
Thanks!

**Comment by t1m0thyj** [comment]
Please verify that the images in the ZIP file exactly match the wildcard "anime_*.jpeg". Could you attach a screenshot of the contents of the ZIP file?

**Comment by ibaeza1** [comment]
![image](https://user-images.githubusercontent.com/37149485/87883841-2e972e00-c9d8-11ea-8ff1-9f5050983785.png)
This is the content of the file. 
I verified that the name of the images its a match with the theme.

**Comment by NelsonDong41** [comment]
I have the same problem but I haven't zipped the folder. Is it required that the folder be zipped?

**Comment by t1m0thyj** [comment]
It is not required to zip the folder. You can also import a JSON file with images alongside it in the same unzipped folder.

Could you please attach one of the themes that is failing to import?

**Comment by ThaiChai** [comment]
I had this same problem and fixed it! When I was making my own folder, I used .jpeg files, since the wiki said to translate .HEIC files to .jpeg. When I wrote my theme.json file, I used the sample files from the wiki as a base and changed them from there.

In my code, which I got from the sample code, the "imageFilename" was set to .jpg instead of. jpeg

For example: "[Name]_*.jpg" instead of "[Name]_*.jpeg"

Once I fixed that it worked for me. Hope that helps!
